### PR TITLE
SG-33500 New build resources yml

### PR DESCRIPTION
**Pull request by eduardoChaucaGallegos**
- New build resources yml

### CODE REVIEW

File: build_resources.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# YAML Configuration File for Qt Resource Building
+#
+# This YAML file defines the configuration for building Qt UI and resource files using the tk-build-qt-resources script.
+# Each entry in the file represents a separate build configuration, to know more about each field please visit
+# https://github.com/shotgunsoftware/tk-toolchain?tab=readme-ov-file#tk-build-qt-resources.
```
Comment (julien-lang) [inline build_resources.yml:5]:
```suggestion
# YAML Configuration File for Qt Resource Building
#
# For more information: https://github.com/shotgunsoftware/tk-toolchain/blob/master/README.md#tk-build-qt-resources
```

File: build_resources.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+- ui_src: resources
+  res_files:
+    - resources
+  py_dest: python/tk_multi_demo/ui
+  import_pattern: tank.platform.qt
```
Comment (julien-lang) [inline build_resources.yml:11]: Remove default values if https://github.com/shotgunsoftware/tk-toolchain/pull/65#discussion_r1698774692 is approved

**Review by julien-lang — COMMENTED** [review]
Looks good. Just a couple minor points before I approve

**Review by julien-lang — APPROVED** [review]

**Review by julien-lang — APPROVED** [review]

File: python/tk_multi_demo/demos/screen_capture_widget/ui/screen_capture_widget_demo.py [hunk 3]
```diff
@@ -1,158 +1,226 @@
...
+        self.dummy_label.setText("")
+    # retranslateUi
+
+Error: C:\Users\chaucae\Documents\projects\tk-multi-demo\python\tk_multi_demo\demos\screen_capture_widget\resources\screen_capture_widget_demo.ui: Warning: Z-order assignment: '' is not a valid widget.
+
+C:\Users\chaucae\Documents\projects\tk-multi-demo\python\tk_multi_demo\demos\screen_capture_widget\resources\screen_capture_widget_demo.ui: Warning: Z-order assignment: 'output_left_spacer' is not a valid widget.
+
+C:\Users\chaucae\Documents\projects\tk-multi-demo\python\tk_multi_demo\demos\screen_capture_widget\resources\screen_capture_widget_demo.ui: Warning: Z-order assignment: 'output_right_spacer' is not a valid widget.
 
+while executing 'c:\users\chaucae\documents\projects\tk-core\.venv\lib\site-packages\PySide2\uic -g python -g python --from-imports C:\Users\chaucae\Documents\projects\tk-multi-demo\python\tk_multi_demo\demos\screen_capture_widget\resources/screen_capture_widget_demo.ui'
```
Comment (carlos-villavicencio-adsk) [inline python/tk_multi_demo/demos/screen_capture_widget/ui/screen_capture_widget_demo.py:226]: These are causing syntax errors.

**Review by carlos-villavicencio-adsk — CHANGES_REQUESTED** [review]

**Review by carlos-villavicencio-adsk — APPROVED** [review]
LGTM

**Pull request merged**